src/test/scala/io/iohk/ethereum/vm/MockWorldState.scala

```diff
@@ -27,6 +29,12 @@ case class MockWorldState(
   def getStorage(address: Address): MockStorage =
     storages.getOrElse(address, MockStorage.Empty)
 
+  def getBlockHash(number: BigInt): Option[ByteString] = if (numberOfHashes >= number && number >= 0) {
+    Some(ByteString(kec256(number.toByteArray)))
+  } else {
+    None
+  }
```

Comment:
Minor. [Scala style guide](http://docs.scala-lang.org/style/control-structures.html#curly-braces) suggest avoiding curly braces in such simple expressions.